### PR TITLE
Insert the 'Slot' to the parser result of LsPci

### PR DESCRIPTION
**Pull request by xiangce**
Signed-off-by: Xiangce Liu <xiangceliu@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

The `Slot` is Key information for each PCI device outputted by the `lspci -k` command.
Typical output of `lspci -k`:
```
]# lspci -k
00:00.0 Host bridge: Intel Corporation 440FX - 82441FX PMC [Natoma] (rev 02)
        Subsystem: Red Hat, Inc. Qemu virtual machine
00:01.0 ISA bridge: Intel Corporation 82371SB PIIX3 ISA [Natoma/Triton II]
        Subsystem: Red Hat, Inc. Qemu virtual machine
```
And since it is named as `Slot` in the output of `lspci -vmmkn`, I renamed the name of the variable `bus_device_function` used in the parser to `slot` too.
Typical output of `lspci -vmmkn`
```
insights_repos]# lspci -vmmkn
Slot:   00:00.0
Class:  0600
Vendor: 8086
Device: 1237
SVendor:        1af4
SDevice:        1100
Rev:    02

Slot:   00:01.0
Class:  0601
Vendor: 8086
Device: 7000
SVendor:        1af4
SDevice:        1100
```

This PR added it to the existing result and won't break the current rules.

### CODE REVIEW

File: insights/parsers/lspci.py [hunk 1]
```diff
@@ -77,21 +79,23 @@ def parse_content(self, content):
             scanner(self)
         # Parse kernel driver lines
         self.data = {}
-        bus_device_function = None
-        bus_device_function_re = re.compile(r'[0-9a-f]+:[0-9a-f]+.[0-9a-f]+')
+        slot = None
+        solt_re = re.compile(r'^[0-9a-f]+:[0-9a-f]+.[0-9a-f]+')
```
Comment (bfahr) [inline insights/parsers/lspci.py:83]: @xiangce I think you meant to use `slot_re` here and below?

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/parsers/lspci.py (hunk 1 above)
Comment (xiangce) [reply insights/parsers/lspci.py]: @bfahr - Fixed. Thanks for reviewing.

**Review by bfahr — COMMENTED** [review]
@xiangce this PR may break backward compatibility.  Could you please review.

**Comment by xiangce** [comment]
> @xiangce this PR may break backward compatibility. Could you please review.

@bfahr - Just fixed the test in the GSS repo, and now all the tests pass.

Please have a check again. Thanks.

**Comment by xiangce** [comment]
@bfahr - It's necessary to merge this PR and the MR to the GSS repo together. I @ed you in the MR. Please merge them together if no other comments. Thanks.

**Pull request merged**